Add note about smart proxy workaround for ACD (#2072)
theforeman/foreman-documentation: guides/common/modules/proc_deploying-an-application-instance.adoc

The firewall settings must be handled by the Ansible playbook itself.
Ensure to open any required network ports for any defined service by creating an additional role or task in the Ansible playbook.
====

[NOTE]
====
ACD detects which {SmartProxyServer} is used to configure the application instance and its hosts.

ifdef::orcharhino[]
image::common/acd-deploying-an-application-instance-rex-target-host-orcharhino.png[Deploying an ACD Application Instance]
endif::[]

For each {SmartProxyServer}, the Ansible playbook runs to configure all hosts.
The remote execution plug-in, which works host-based, is used to run the Ansible playbook on each {SmartProxyServer}.

> Therefore, the first host of the application instance runs the Ansible playbook.

Author: No worries. Applied.

> Therefore, the first host of the application instance is used to initiate running the Ansible playbook.

Reviewer: Maybe?
```suggestion
Therefore, the first host of the application instance runs the Ansible playbook.
```

Author: Applied.

The *configuration jobs* page shows the first host of each {SmartProxyServer} as target host.

> The ACD component of the {SmartProxy} starts an Ansible playbook which configures all hosts connected to this {SmartProxyServer}.

Reviewer: ```suggestion
The ACD component of {SmartProxy} starts an Ansible playbook which configures all hosts connected to this {SmartProxyServer}.
```

Author: Applied.
